### PR TITLE
Hot plugging

### PR DESCRIPTION
**Pull request by danellenachum**
Added hot plugging for sensors

### CODE REVIEW

File: runtime/runtime.py [hunk 1]
```diff
@@ -523,6 +575,7 @@ def send_motor_data(data):
     })
 
     # Update sensor values, and send to UI
+    update_hibike()
```
Comment (nikitakit) [inline runtime/runtime.py:598]: Did you intend to also call update_motors()

Comment (danellenachum) [inline runtime/runtime.py:598]: No, that function doesn't work. I actually meant to delete it altogether.

Comment (matthew-zhao) [inline runtime/runtime.py:598]: Are we still planning to add hot plugging for motors? We don't have that much time left lol

Comment (danellenachum) [inline runtime/runtime.py:598]: I could try going to 101 tomorrow to test/tweak the code I already wrote to see if I could get it to work.

Comment (matthew-zhao) [inline runtime/runtime.py:598]: Ok Danelle, let me know if you have issues, and tell me when you leave whether you are going to tweak the code or not. I won't be in 101 today, but hopefully someone is to help you.

**Comment by matthew-zhao** [comment]
Hey Danelle,

I've merged in Bob's encoder fix. That seemed to have caused conflicts on this branch. To resolve, pull from pioneers fork (upstream) develop branch and then fix the merge conflicts (do you know how to do this?), then push to your own (origin) develop branch

**Pull request merged**